.github/workflows/run_specs.yml

```diff
@@ -0,0 +1,57 @@
+env:
+  RUBY_VERSION: 3.0.0
+  POSTGRES_PASSWORD: postgres
+  DEVISE_JWT_SECRET_KEY: ${{ secrets.DEVISE_JWT_SECRET_KEY }}
+
+name: Rails Specs
+on: [push]
+jobs:
+  rspec-test:
+    name: Rubocop/RSpec
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:latest
```

Comment:
could you specify version, please? Don't use the latest for this case. DB should be the same everywhere.